src/status_im/utils/money.cljs

```diff
@@ -22,10 +23,27 @@
 (defn bignumber [n]
   (dependencies/Web3.prototype.toBigNumber (str n)))
 
-(def ether-unit-value (bignumber "1000000000000000000"))
+(def eth-units
+  {:wei    (bignumber "1")
+   :kwei   (bignumber "1000")
+   :mwei   (bignumber "1000000")
+   :gwei   (bignumber "1000000000")
+   :szabo  (bignumber "1000000000000")
+   :finney (bignumber "1000000000000000")
+   :eth    (bignumber "1000000000000000000")
+   :keth   (bignumber "1000000000000000000000")
+   :meth   (bignumber "1000000000000000000000000")
+   :geth   (bignumber "1000000000000000000000000000")
+   :teth   (bignumber "1000000000000000000000000000000")})
+
+(defn wei-> [unit n]
```

Comment:
Can we throw something if it can't find unit? Like a pre-condition or something. So we don't accidentally typo unit or someone passes in `":geth"` or whatever and we get nonsense back.

Reply:
yes at some point I also think this could be improved further by having maps as value with alternative symbols, string symbols and other things

Reply:
You already get 'not a number null in wei->' when putting the wrong unit though isn't it enough considering it's a developer only interface ?

Reply:
If it already throws an error whenever unit is not found then OK